packages/transmute-services/test/Oracle/interface.truffle.js

```diff
@@ -50,16 +68,23 @@ contract("works with framework", accounts => {
       TRANSMUTE_API_ROOT: "http://localhost:3001"
     });
 
-    factory = await OracleFactory.deployed();
+    oracle = await Oracle.deployed();
     oracleCaller = await OracleCaller.deployed();
+
+    listenForRequest(T, accounts, oracle, oracleCaller);
   });
 
-  it("OracleFactory can create an oracle", async () => {
-    let { events, tx } = await T.Factory.createEventStore(factory, accounts[0]);
-    oracleAddress = events[0].payload.address;
-    oracle = await Oracle.at(oracleAddress);
+  it("protocol", async () => {
+    // trigger caller, caller will make a request
+    await oracleCaller.trigger({ from: accounts[0] });
   });
 
+  // it("OracleFactory can create an oracle", async () => {
```

Comment:
these dead tests should be removed, they are handled above. maybe add some comments regarding what the test cases that are covered by that event listener pattern.

Reply:
They weren't removed because I intend to reimplement them and the comments serve as a reminder - will add comments as well.